dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -8,6 +8,9 @@ RUN apt-get install -f -y ifupdown bridge-utils libdbus-1-3 libdaemon0 libjansso
 ## TODO: implicitly install dependencies
 RUN apt-get -y install libjemalloc1
 
+# Dependencies for sonic-cfggen                                                                                                                                                           
+RUN apt-get install -y python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml 
```

Comment:
We' don't really have a timing to remove them (as config is done upon entrypoint).
And leaving them there will give us the potential to update config later.

---
In reply to: [99409145](https://github.com/Azure/sonic-buildimage/pull/238#discussion_r99409145) [](ancestors = 99409145)